fe1-web/src/features/witness/README.md

```diff
@@ -0,0 +1,16 @@
+# Witnessing
+
+Should provide the witness functionality. Currently only supports witness
```

Comment:
When you use "currently", it could be nice to add a timing context like the current date so that we know when it was written without having to check GitHub.

Reply:
True 👍